### DISCUSSION TITLE
Arm docker image

### DISCUSSION THREAD

**Discussion by Casburggraaf** [outline/outline#2712] [Self Hosting]
Is there an arm(64) image available for outline

**Comment by tommoor** [comment]
There is no official image for ARM

**Comment by Snuupy** [comment]
I looked into this out of curiosity, I think you can build your own ARM images as none of the dependencies require x86. Redis, postgres, nginx, etc. are all supported on ARM so only the outline docker image would need to be built. :)

**Comment by sebgeekinside** [comment]
The image outlinewiki/outline has the tags `linux/arm/v7` and `linux/arm64` on docker hub but when I launch the latest docker image on a Raspberry Pi (ARM), the following message is raised: `exec /usr/local/bin/docker-entrypoint.sh: exec format error`.

Is this docker image realy ARM architecture compatible?

I see into the Dokerfile that this docker image is based on the outlinewiki/outline-base that has only the tag `linux/amd64` on docker hub...
